store/migrations/migrate.go

```diff
@@ -72,6 +73,11 @@ func Migrate(db *gorm.DB) error {
 			Migrate:  migration1605288480.Migrate,
 			Rollback: migration1605288480.Rollback,
 		},
+		{
+			ID:       "1607327594",
+			Migrate:  migration1607327594.Migrate,
+			Rollback: migration1607327594.Rollback,
+		},
```

Comment:
When rebasing, this migration will probably need to be updated to the current timestamp to be placed in the end of the queue.